primitives/arithmetic/fuzzer/src/fixed_point.rs

```diff
@@ -16,19 +16,19 @@
 // limitations under the License.
 
 //! # Running
-//! Running this fuzzer can be done with `cargo hfuzz run fixed`. `honggfuzz` CLI options can
+//! Running this fuzzer can be done with `cargo hfuzz run fixed_point`. `honggfuzz` CLI options can
```

Comment:
A follow up that you can consider is extending these to other types as well. It seems like now we only fuzz Fixed64, while polkadot uses Fixed128 a lot.